architecture/core_concepts/pods_and_services.adoc

```diff
@@ -153,12 +153,19 @@ has details about the functionality and purpose of pods.
 
 == Services
 
-A Kubernetes `*service*` serves as an internal load balancer. It identifies
-a set of replicated xref:pods[pods] in order to proxy the connections it
+A Kubernetes link:http://kubernetes.io/docs/user-guide/services#[service] serves as an internal load balancer.
+It identifies a set of replicated xref:pods[pods] in order to proxy the connections it
 receives to them. Backing pods can be added to or removed from a service
 arbitrarily while the service remains consistently available, enabling
 anything that depends on the service to refer to it at a consistent
-internal address.
+address.
+
+The default service IP addresses are from the {product-title} internal network and are used to permit pods to
```

Comment:
Careful.  There are two different IP ranges.  The "service IP" must be in the private "service network" which is distinct from the pod SDN network.

And a service can have two different IP addresses, the "service IP" and the "external IP".